source/deprecations/v3.x.html.md

```diff
@@ -53,3 +53,57 @@ object.notifyPropertyChange('someProperty');
 ##### id: ember-metal.getting-each
 
 Calling `array.get('@each')` is deprecated. `@each` may only be used as dependency key.
+
+### Deprecations Added in Upcoming Versions
```

Comment:
Sorry for the conflicting advice, can you change this to be within the 3.2 section? I'm pretty sure we will be able to tackle it before 3.2 goes to beta...